src/OpenLoco/Entities/EntityManager.cpp

```diff
@@ -136,13 +137,27 @@ namespace OpenLoco::EntityManager
         else
         {
             auto* previousEntity = get<EntityBase>(previousId);
-            previousEntity->next_thing_id = nextId;
+            if (previousEntity == nullptr)
+            {
+                std::cerr << "Invalid previous entity id. Entity linked list corrupted?";
```

Comment:
I think it would be best to stick with `Console::error` for this.

Reply:
Argh I couldn't find what we were using for errors. I could only remember log_error from OpenRCT2.